src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/config/HdcedConfiguration.kt

```diff
@@ -9,4 +9,6 @@ data class HdcedConfiguration(
   val custodialPeriodMidPointDays: Long,
   val custodialPeriodAboveMidpointDeductionDays: Long,
   val custodialPeriodBelowMidpointMinimumDeductionDays: Long,
+  val custodialPeriodMidPointDaysHdc365: Long,
```

Comment:
For long term values it would be better to have "Classic HDC" or "Legacy HDC" and HDC365 can just be HDC - that way refactoring old code can target Legacy / Classic rather than leaving or repairing the new code